config/locales/en.yml

```diff
@@ -323,24 +320,28 @@ en:
   categories:
     add_category: Add category
     add_psid: Add PetScan PSID
+    add_pileid: Add PagePile id
     add_template: Add template
     add_this_category: Add this category
     add_this_psid: Add this PSID
+    add_this_pileid: Add this Page pile id
```

Comment:
Use consistent terminology for PagePile ID in all interface message. I think "PagePile ID" is the best option.

Reply:
Please update all the messages to "PagePile ID".